fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -74,6 +74,22 @@ unsigned EligibilityCounter::getCount(int combinedType) const {
 
 } // namespace data_distribution
 
+size_t getNChooseKLowerBound(size_t n, size_t k) {
+	if (k > n)
+		return 0;
+	if (k * 2 > n)
+		k = n - k;
```

Comment:
can you move this below the case `k == 0`. Almost confused to think returning `n -k ` here.